src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -1834,23 +1834,23 @@ public sealed partial class MTAThreadAttribute : System.Attribute
     }
     public abstract partial class MulticastDelegate : System.Delegate
     {
-        protected MulticastDelegate(object target, string method) : base (default(object), default(string)) { }
-        protected MulticastDelegate(System.Type target, string method) : base (default(object), default(string)) { }
-        protected sealed override System.Delegate CombineImpl(System.Delegate follow) { throw null; }
-        public sealed override bool Equals(object obj) { throw null; }
+        protected MulticastDelegate(object target, string method) : base (default(object)!, default(string)!) { }
```

Comment:
For these cases I didn't know if I should add a ! or disable the warning for reference assemblies. This would give: `error CS8625: Cannot convert null literal to non-nullable reference type.` because the base constructor doesn't accept nulls. 


We could also add this into GenAPI itself, but don't know how much work it would be to actually figure out if the referenced constructor can actually accept null or not for every parameter. It seems doable but hasn't investigated yet.

Thoughts?

Reply:
I suggest we disable nullable warnings for reference assemblies, or at least the warnings that have to do with implementation (there might be a little value in warnings that have to do with validating annotations in signatures are correct, such as the relationship between annotations in overrides), since the implementation in reference assemblies are all placeholders.

Reply:
That is what I ended up doing since I don't think it is common to have a null literal in reference assemblies. I could only think of Attributes, Fields and Base calls. However I never expect we would declare an Attribute with a null literal in implementation and if a public Field has a null value, I would expect its type to be nullable.